crates/sozo/src/commands/events.rs

```diff
@@ -0,0 +1,51 @@
+use crate::ops::events;
+
+use super::options::{
+    dojo_metadata_from_workspace, starknet::StarknetOptions, world::WorldOptions,
+};
+use anyhow::Result;
+use clap::Args;
+use scarb::core::Config;
+
+#[derive(Args, Debug)]
+pub struct EventsArgs {
+    #[clap(short, long)]
+    #[clap(help = "Number of events to return per page")]
+    pub chunk_size: u64,
+
+    #[clap(short, long)]
+    #[clap(help = "Block number from where to look for events")]
+    pub from_block: Option<u64>,
+
+    #[clap(short, long)]
+    #[clap(help = "Block number until where to look for events")]
+    pub to_block: Option<u64>,
+
+    #[clap(short, long)]
```

Comment:
Imo it might be better if we make it into positional args, so that we don't need to do `--events`. wdyt @broody ? Makes the command much easier to use.

Reply:
Agreed!

Reply:
by positional arg do you mean this https://docs.rs/clap/latest/clap/_cookbook/escaped_positional_derive/index.html?

then we would need to specify all events after `--`
> sozo events -c 1 --rpc-url http://localhost:5050  --  ComponentRegistered,PlayerJoined

Reply:
simply removing the `#[clap(short,long]` will turn it into positional arg

Reply:
oh TIL